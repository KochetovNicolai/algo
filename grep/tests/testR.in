[^0-9A-Z]+
ABC139
AaBbCc
AbcD
0123456789
`!@#$%^&*()
