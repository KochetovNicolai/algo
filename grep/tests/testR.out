0	
0	
0	
0	
0	
0	

0	
1	2 
0	
1	4 
0	
1	6 

0	
2	2 3 
1	3 
0	

0	
0	
0	
0	
0	
0	
0	
0	
0	
0	

11	1 2 3 4 5 6 7 8 9 10 11 
10	2 3 4 5 6 7 8 9 10 11 
9	3 4 5 6 7 8 9 10 11 
8	4 5 6 7 8 9 10 11 
7	5 6 7 8 9 10 11 
6	6 7 8 9 10 11 
5	7 8 9 10 11 
4	8 9 10 11 
3	9 10 11 
2	10 11 
1	11 

